plugins/scaffolder-node/api-report.md

```diff
@@ -256,6 +256,9 @@ export const parseRepoUrl: (
   project?: string | undefined;
 };
 
+// @public
+export const restoreWorkspace: (path: string, buffer?: Buffer) => Promise<void>;
```

Comment:
Thinking that we should wrap these up in objects.

`restoreWorkspace: (opts: { path: string, buffer?: Buffer }) => Promise<void>`

`serializeWorkspace: (opts: { path: string }) => Promise<{ contents: Buffer }>`

Reply:
Actually, a thought here, maybe the interface for the `WorkspaceProvider` should be altered slightly. 

Maybe `restoreWorkspace` and `serializeWorkspace` should actually happen in the [/StorageTaskBroker.ts](https://github.com/backstage/backstage/pull/25142/files#diff-6282ab375a8d05e3300ecada8c839fd1045b49c5fc6eb9ea10830a9bc1fc1b3d) instead, rather than having all the providers having to use these methods?

What do you think?

Reply:
Hey 👋 I was thinking about that, but then remembered the comment from @freben here https://github.com/backstage/backstage/pull/24200#discussion_r1584590573 😅

Reply:
Ah ok - In that case, let's move them to `alpha` exports and wrap up args and return in object?